Docs/Configuration.tex

```diff
@@ -6103,8 +6138,8 @@ \subsection{Audio Properties}\label{uefiaudioprops}
   \textbf{Description}: Audio codec reconfiguration delay in microseconds.
 
   Some codecs require a vendor-specific delay after the reconfiguration
-  (e.g. volume setting). This option makes it configurable. In general
-  the necessary delay may be as long as 0.5 seconds.
+  (e.g. volume setting). This option makes it configurable. The required
+  delay may be as long as 0.5 seconds.
```

Comment:
It changes the meaning here. We observed the issue on systems that require 0.5 seconds. But it does not mean all the systems will have this delay. This value is rather a common-sense recommendation.